### ISSUE TITLE
include_vars

### ISSUE THREAD

**Issue by martbhell** [lxsli/ansible-viz#2]
Hello, 

I'm using a quite a few roles which have include_vars: in them.
This seems to not work and gives errors like this:

<pre>
Problem resolving task 'ansible-role-ntp::main' include_vars: '{{'
/home/me/Git2/ansible-viz/resolver.rb:98:in `block in resolve_task_include_vars': Unhandled include_vars: {{ (RuntimeError)
        from /home/me/Git2/ansible-viz/resolver.rb:87:in `map!'
        from /home/me/Git2/ansible-viz/resolver.rb:87:in `resolve_task_include_vars'
        from /home/me/Git2/ansible-viz/resolver.rb:22:in `block in process'
        from /home/me/Git2/ansible-viz/resolver.rb:20:in `each'
        from /home/me/Git2/ansible-viz/resolver.rb:20:in `process'
        from ./ansible-viz.rb:60:in `render'
        from ./ansible-viz.rb:100:in `<main>'
</pre>

https://github.com/resmo/ansible-role-ntp

**Comment by elzibubble** [comment]
Hi there, I'm sorry but I'm no longer interested in working on this. It was briefly useful before a) the Ansible became too hard to parse, b) we established conventions in our playbooks that made them mostly comprehensible without a tool like this. If you want to fork, please go ahead - just be warned the code is rather noodly.
